app/lang/de/dashboard.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+return [
+
+    'metro' => [
+        'players_online' => 'Spieler Online',
+        'average_bans' => 'Durchschnittliche Banns pro Tag',
+        'yesterday_bans' => 'Banns gestern',
+        'player_count' => 'Spezifische Spieler',
+        'adkats' => [
+            'titles' => [
+                'killed' => 'Getötete Spieler',
+                'kicked' => 'Gekickte Spieler',
+                'banned' => 'Gebannte Spieler',
+                'banned_active' => 'Aktive Banns'
+            ],
+            'killed' => ':%killed der Spieler wurden getötet',
```

Comment:
@BP4U The % sign has to be in front of the colon.

So it should be ```%:killed```. Fix that then push back up and let me know when you did. The pull request should reflect those changes once you sync.